src/v0/destinations/fb_custom_audience/transform.js

```diff
@@ -378,10 +378,7 @@ const processRouterDest = (inputs, reqMetadata) => {
         return getSuccessRespEvents(input.message, [input.metadata], input.destination);
       }
       const transformedList = process(input);
-      const responseList = transformedList.map((transformedPayload) =>
-        getSuccessRespEvents(transformedPayload, [input.metadata], input.destination),
-      );
-      return responseList;
+      return getSuccessRespEvents(transformedList, [input.metadata], input.destination);
```

Comment:
I think we can remove this code in processRouterDest and just use `simpleProcessRouterDest`. We're not doing anything extra to what `simpleProcessRouterDest` does

Reply:
Yes updated the code